### PR TITLE
Tournaments/change notification model into announcement [V1 -- Outdated]

### PR DESCRIPTION
**Pull request by MuyanXie**
PR description for Issue #162 

Overview:

- We tried to the models to use instead an announcement model
- We imported and used the notification model from the User's team
- We provided a method to send notifications by auto-generating notifications to the correct receiver

Key changes:

- deleted the notification model in games views
- added announcement model for games views using notification from user views
- merged discrepancies in database scheme

### CODE REVIEW

**Comment by MuyanXie** [comment]
@MikeChen012345 Could you please take a look at this as well?

**Comment by MuyanXie** [comment]
Found the user's team response to the notification model @MikeChen012345 :

https://github.com/uchicago-cs/chigame/wiki/Users-&-Friending-~-Notification-Model-Documentation

File: src/chigame/games/models.py [hunk 1]
```diff
@@ -257,6 +276,9 @@ class Message(models.Model):
     sender = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
     # Even if the user is deleted, the message will still exist such that the
     # message history is preserved. The sender field will be set to null.
+    sender = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
```
Comment (elizabethli31) [inline src/chigame/games/models.py:279]: Two senders (repeated code - delete one)

File: src/chigame/games/models.py [hunk 2]
```diff
@@ -207,31 +207,50 @@ def __str__(self):  # may be changed later
...
 
 
-class Notification(models.Model):
+class Announcement(models.Model):
     """
-    A notification, which can be sent to multiple users.
+    An announcement, which can be sent to multiple users.
     """
 
     recipients = models.ManyToManyField(User, related_name="notifications")
```
Comment (elizabethli31) [inline src/chigame/games/models.py:215]: Delete this one on line 215.

**Review by elizabethli31 — CHANGES_REQUESTED** [review]
Generally looks good, but you have some repeats that need to be deleted. Also, please make this PR **ONLY** changes for the notification/announcement model. You have changes from your other PRs on here (all the CRUD views) which should be in the other branches. Delete those files or remove them from git for this branch. Also, I think you are not up to date with dev which has an 0006 migration, so you might need to delete this one, and re-migrate.

**Comment by MuyanXie** [comment]
This PR can be safely closed as I made a new PR based on the new Dev which contained most of our model logic there

**Pull request closed without merging**